src/Native/Unix/System.Security.Cryptography.Native/opensslshim.h

```diff
@@ -831,6 +832,7 @@ FOR_ALL_OPENSSL_FUNCTIONS
 #define SSL_free SSL_free_ptr
 #define SSL_get_client_CA_list SSL_get_client_CA_list_ptr
 #define SSL_get_current_cipher SSL_get_current_cipher_ptr
+#define SSL_CIPHER_get_id SSL_CIPHER_get_id_ptr
```

Comment:
`SSL_CIPHER_`-stuff is up around line 811.

Reply:
i wonder if this wasn't rebase issue - I'm almost sure I've put them together before...